### PR TITLE
Green CI

### PR DESCRIPTION
**Pull request by HypeMC**
The CI is in a bit of a mess, hope this helps.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -194,7 +195,7 @@ jobs:
               run: vendor/bin/simple-phpunit
 
     phpunit-macos:
-        runs-on: macos-latest
+        runs-on: macos-13
```
Comment (HypeMC) [inline .github/workflows/ci.yml:198]: macos-latest doesn't include the geckodriver

Comment (dunglas) [reply .github/workflows/ci.yml:198]: If necessary, it is available on brew (which is preinstalled): https://formulae.brew.sh/formula/geckodriver

**Review by dunglas — APPROVED** [review]
Thank you!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (HypeMC) [reply .github/workflows/ci.yml]: @dunglas Done, I had to install both Firefox & geckodriver, but it seems to be working now. However, it's kind of slow, over 15 min.

**Comment by dunglas** [comment]
Thanks!

**Pull request merged**